server/session.go

```diff
@@ -610,3 +623,12 @@ func (p *Plugin) getSessionByOriginalID(sessionID string) *session {
 
 	return nil
 }
+
+func (p *Plugin) hasSessionsForCall(callID string) bool {
+	for _, s := range p.sessions {
+		if s.callID == callID {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
This is fine for now. At some point, it may be worth mapping sessions to their call ID.